common/sass/styles/overrides/react-select.scss

```diff
@@ -0,0 +1,62 @@
+// This syntax is necessary to override css styles in react-select
+.Select {
+  font-size: 1rem;
+  .Select-control {
```

Comment:
Select controls are currently 38px with border. Other inputs are 40.8px with border at full size, using the `$input-height-base` variable. We should try to line these up so they don't look awkward when next to each other:
![offset](https://user-images.githubusercontent.com/649992/35294419-85193cec-0044-11e8-9949-8fdbc1843d83.png)